informer/informer_integration_test.go

```diff
@@ -0,0 +1,194 @@
+// +build integration
+
+package informer
+
+/*
+	Usage:
+
+		go test -tags=integration ./informer [FLAGS]
+
+	Flags:
+
+		-integration.address string
+			Kubernetes API server address (default "https://$(minikube ip):8443")
+		-integration.ca string
+			CA file path (default "$HOME/.minikube/ca.crt")
+		-integration.crt string
+			certificate file path (default "$HOME/.minikube/apiserver.crt")
+		-integration.key string
+			key file path (default "$HOME/.minikube/apiserver.key")
+*/
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os/exec"
+	"os/user"
+	"path"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/cenk/backoff"
+	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/watch"
+
+	"github.com/giantswarm/micrologger/microloggertest"
+	"github.com/giantswarm/operatorkit/client/crdclient"
+	"github.com/giantswarm/operatorkit/crd"
+	fakecrd "github.com/giantswarm/operatorkit/crd/fake"
+)
+
+var (
+	address string
+
+	caFile  string
+	crtFile string
+	keyFile string
+
+	newCRD            *crd.CRD
+	newCRDClient      apiextensionsclient.Interface
+	newInformer       *Informer
+	newWatcherFactory WatcherFactory
+)
+
+func init() {
+	var err error
+
+	{
+		u, err := user.Current()
+		homePath := func(relativePath string) string {
+			if err != nil {
+				return ""
+			}
+			return path.Join(u.HomeDir, relativePath)
+		}
+
+		var serverDefault string
+		{
+			out, err := exec.Command("minikube", "ip").Output()
```

Comment:
With `shipyard` you could spin up a remote minikube in aws and run these tests on circleCI, let me know if i can help with that :)

Reply:
I want this too :D

Reply:
100%, I pinged you to get feedback on this. I would like to get the PR done first and integrate it into Circle separately. Just let me know how this works.